packages/RichTextEditing-Core.package/TextStructure.class/instance/add..st

```diff
@@ -0,0 +1,6 @@
+adding
+add: aTextAttribute
+
+	(self attributes includes: aTextAttribute)
+		ifTrue: [self remove: aTextAttribute].
```

Comment:
There should be some explanation somewhere, why we remove something before adding it

Reply:
I will explain it in the class comment of TextStructure.